### PR TITLE
feat(sdk): Show `len` in `Runs.__repr__`

### PR DESCRIPTION
**Pull request by janosh**

### CODE REVIEW

**Comment by janosh** [comment]
@gtarpenning Could you have a quick look at this?

**Comment by gtarpenning** [comment]
> @gtarpenning Could you have a quick look at this?

Seems reasonable, we can probably get this into the next major python-package release :) Do you care if your name is attached to it?

**Comment by janosh** [comment]
> Do you care if your name is attached to it? @janosh

I don't think such a small change warrants that but I don't mind.

File: wandb/apis/public.py [hunk 1]
```diff
@@ -1657,8 +1657,8 @@ def convert_objects(self):
 
         return objs
 
-    def __repr__(self):
-        return f"<Runs {self.entity}/{self.project}>"
+    def __repr__(self) -> str:
+        return f"<Runs: {len(self)} in {self.entity}/{self.project}>"
```
Comment (gtarpenning) [inline wandb/apis/public.py:1661]:
```suggestion
        return f"<Runs {self.entity}/{self.project} ({len(self)})>"
```

Comment (gtarpenning) [reply wandb/apis/public.py:1661]: I think this styling is slightly preferable

**Review by gtarpenning — CHANGES_REQUESTED** [review]
lets just append the run count in parenthesis. Like:

`Runs janosh/testing-project (292)`

**Comment by janosh** [comment]
@gtarpenning Anything more needed here?

**Pull request closed without merging**